### ISSUE TITLE
Link to 2013.3 FBX exporter missing in PSK/FBX to PCC tutorial

### ISSUE THREAD

**Issue by MaZT3Rz** [Mgamerz/ME3Explorer#338]
Self explanatory. It just takes you to Discord with nothing in it.

https://github.com/ME3Tweaks/LegendaryExplorer/wiki/How-to-get-your-meshes-setup-for-import-into-a-PCC

**Comment by Mgamerz** [comment]
Where is this tutorial?

**Comment by MaZT3Rz** [comment]
https://github.com/ME3Tweaks/LegendaryExplorer/wiki/How-to-get-your-meshes-setup-for-import-into-a-PCC

**Comment by Mgamerz** [comment]
![Screenshot_20220906-142315.png](https://user-images.githubusercontent.com/2738836/188731110-a983d8b1-6753-44e6-9f9e-0f77e397e85f.png)

Seems to work for me? It went right to it. It should probably be hosted on not discord though.

**Comment by MaZT3Rz** [comment]
![Näyttökuva 2022-09-06 232507](https://user-images.githubusercontent.com/88979692/188731461-ee8e8f63-314b-4620-bbbc-42dfbf86e230.png)

**Comment by MaZT3Rz** [comment]
Anyway, do you have the link to the script?

**Comment by Mgamerz** [comment]
The link has been updated in the wiki.

**Comment by MaZT3Rz** [comment]
Thanks, now the link works, but there's a problem, I'm getting this when exporting anything in FBX with the new script:

```
Python: Traceback (most recent call last):
  File "C:\Program Files (x86)\Steam\steamapps\common\Blender\3.3\scripts\addons\io_scene_fbx\__init__.py", line 636, in execute
    return export_fbx_bin.save(self, context, **keywords)
  File "C:\Program Files (x86)\Steam\steamapps\common\Blender\3.3\scripts\addons\io_scene_fbx\export_fbx_bin.py", line 3198, in save
    ret = save_single(operator, context.scene, depsgraph, filepath, **kwargs_mod)
  File "C:\Program Files (x86)\Steam\steamapps\common\Blender\3.3\scripts\addons\io_scene_fbx\export_fbx_bin.py", line 3061, in save_single
    settings = FBXExportSettings(
TypeError: FBXExportSettings.__new__() missing 1 required positional argument: 'use_custom_props'
```

Edit: Fix for this was to install  2.93 and use the script there.
